lib/api/core/validation/index.js

```diff
@@ -836,13 +820,14 @@ function manageErrorMessage(errorContext, errorHolder, message, structured) {
  * @returns {Promise}
  */
 function getValidationConfiguration(kuzzle) {
-
   return kuzzle.internalEngine
     .search('validations', false, {from: 0, size: 1000})
     .then(result => {
-      var validation = {};
+      let validation;
 
       if (result && result.hits && Array.isArray(result.hits) && result.hits.length > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 90 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)